designer/server/src/main/scala/pl/touk/nussknacker/ui/config/FeatureTogglesConfig.scala

```diff
@@ -50,6 +51,7 @@ object FeatureTogglesConfig extends LazyLogging{
     val intervalTimeSettings = config.as[IntervalTimeSettings]("intervalTimeSettings")
     val testDataSettings = config.as[TestDataSettings]("testDataSettings")
     val redirectAfterArchive = config.getAs[Boolean]("redirectAfterArchive").getOrElse(true)
+    val codeSuggestionsFromBackend = config.getAs[Boolean]("codeSuggestionsFromBackend").getOrElse(false)
```

Comment:
Can you remove this `getOrElse` and add the default into `defaultDesignerConfig.conf` instead? Also let make it enableable by env variable - take a look other examples in this file

Reply:
Done